stubs/regex/regex/regex.pyi

```diff
@@ -568,7 +568,7 @@ class Pattern(Generic[AnyStr]):
         timeout: float | None = None,
     ) -> _regex.Scanner[bytes]: ...
     def __copy__(self) -> Self: ...
-    def __deepcopy__(self) -> Self: ...
+    def __deepcopy__(self, /, memo: Unused) -> Self: ...
```

Comment:
same here, `memo` appears to be positional-only:

```pycon
>>> regex.compile('foo').__deepcopy__(memo=None)
Traceback (most recent call last):
  File "<python-input-11>", line 1, in <module>
    regex.compile('foo').__deepcopy__(memo=None)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
TypeError: Pattern.__deepcopy__() takes no keyword arguments
>>> regex.compile('foo').__deepcopy__(None)
regex.Regex('foo', flags=regex.V0)
```

Reply:
```suggestion
    def __deepcopy__(self, memo: Unused, /) -> Self: ...
```